react-frontend-app/Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV PATH=/app/node_modules/.bin:$PATH
 # install app dependencies
 COPY package.json ./
 RUN npm install
+RUN npm install d3 d3-axis d3-format d3-scale d3-selection
```

Comment:
Rather than installing these directly through the command line, can you add them to the package.json file? 

Also, please include versions of each of the packages, that way the app won't break if the version gets a new release.